examples/scaling-criteo/02-ETL-with-NVTabular.ipynb

```diff
@@ -351,7 +351,11 @@
     "categorify_op = Categorify(out_path=stats_path, max_size=num_buckets)\n",
```

Comment:
<p>Actually, we do not use Frequency thresholding in the NVT pipeline below, therefore we do not set the value to <code>15</code>.I'd recommend you to remove Frequency Thresholding section, and add  couple sentences regarding  what <code>max_size</code> param in the Categorify op does. you can refer to https://github.com/NVIDIA-Merlin/NVTabular/blob/main/nvtabular/ops/categorify.py#L172</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/Merlin/pull/380/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>